<?xml version="1.0"?>
<launch>
    <!-- ===== Launch lidar ===== -->
    <!-- <node name="rplidarNode"          pkg="rplidar_ros"  type="rplidarNode" output="screen">
        <param name="serial_port"           type="string" value="/dev/ttyUSB0"/>
        <param name="serial_baudrate"       type="int"    value="115200"/>
        <param name="frame_id"              type="string" value="laser"/>
        <param name="inverted"              type="bool"   value="false"/>
        <param name="angle_compensate"      type="bool"   value="true"/>
        <param name= "respawn"              type="bool"   value="true"/>
    </node> -->


    <node name="occupacy_grid" pkg="vision_nodes" type="laser_map.py" output="screen">
        <param name="node_path" type="string" value="$(find vision_nodes)" />
        <!--Metodo1: Cargar desde archivo de configuracion-->
        <rosparam command="load" file="$(find vision_nodes)/config/params_grid.yaml"/>
    </node>

</launch>